samples/matter/lock/src/app_task.cpp

```diff
@@ -274,6 +288,71 @@ void AppTask::LockActionEventHandler(const AppEvent &event)
 	}
 }
 
+#ifdef CONFIG_THREAD_WIFI_SWITCHING
```

Comment:
Are we going to introduce the `CONFIG_THREAD_WIFI_SWITCHING` as KConfig and make it dependent on both `CONFIG_NET_L2_OPENTHREAD` and `CONFIG_CHIP_WIFI`? BTW. Is it currently possible to build the application with both WiFi and Thread support? I guess we need to build both separately, right?

Reply:
Yes, we need to build two separate targets. Such config already exists in this sample Kconfig file.